packages/test-base/build.gradle.kts

```diff
@@ -180,41 +176,28 @@ kotlin {
 }
 
 kotlin {
-    // define targets depending on the host platform (Apple or Intel)
     if(System.getProperty("os.arch") == "aarch64") {
         iosSimulatorArm64("ios")
         macosArm64("macos")
     } else if(System.getProperty("os.arch") == "x86_64") {
         iosX64("ios")
         macosX64("macos")
     }
-
     sourceSets {
-        val macosMain by getting
-        val macosTest by getting
-        getByName("iosMain") {
-            kotlin.srcDir("src/macosMain/kotlin")
-        }
-        getByName("iosTest") {
-            kotlin.srcDir("src/macosTest/kotlin")
+        val commonMain by getting
+        val commonTest by getting
+        val nativeDarwin by creating {
+            dependsOn(commonMain)
         }
-    }
-}
-
-// Needs running emulator
-tasks.named("iosTest") {
```

Comment:
So the emulator is started automatically for us now?

Reply:
Yep. Handled by the Kotlin gradle plugin ... the output after removing this is 
```
> Task :test-base:iosX64Test
Starting process 'command '/usr/bin/xcrun''. Working directory: /Users/claus.rorbech/proj/realm-kotlin/packages/test-base Command: /usr/bin/xcrun simctl spawn --standalone iPhone 12 /Users/claus.rorbech/proj/realm-kotlin/packages/test-base/build/bin/iosX64/debugTest/test.kexe -- --ktest_no_exit_code --ktest_logger=TEAMCITY
```